roles/prometheus/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import os
+import yaml
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.fixture()
+def AnsibleDefaults():
+    with open("defaults/main.yml", 'r') as stream:
+        return yaml.safe_load(stream)
```

Comment:
this is the actual change, the rest are just files been added to the repository

Reply:
outdated, we are removing this submodule since we do not use this submodule for our prometheus setup in l-cloud-gcc